switchmap/poller/update/device.py

```
@@ -143,9 +143,22 @@ def process(self):
+                # github.com/PalisadoesFoundation/switchmap-ng/issues/127
+
+                try:
+                    higherlayers = updated_device_data["system"]["IF-MIB"][
+                        "ifStackStatus"
+                    ][ifindex]
+                    log_message = (
+                        f'''\
+Host {self._devicename} is missing ifindex {ifindex} in \
+the IF-MIB::ifStackStatus OID''')
+                    log.log2debug(1071, log_message)
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)